Source/Core/Core/ConfigManager.cpp

```diff
@@ -819,10 +822,11 @@ const char* SConfig::GetDirectoryForRegion(DiscIO::Region region)
 
   case DiscIO::Region::NTSC_K:
     ASSERT_MSG(BOOT, false, "NTSC-K is not a valid GameCube region");
-    return nullptr;
+    return JAP_DIR;  // See ToGameCubeRegion
 
   default:
-    return nullptr;
+    // This should never be reached. If it is, some cases are missing.
```

Comment:
If this never should be reached, put an assert here, like with the NTSC-K case.

Reply:
Ah, of course.